src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposePull.groovy

```diff
@@ -19,15 +31,12 @@ class ComposePull extends DefaultTask {
 
     @TaskAction
     void pull() {
-        if (settings.buildBeforePull.get()) {
```

Comment:
Is there any replacement for this feature in this PR? 🙏 

I'm OK with removal of this feature (because it makes more sense to use standard Gradle tasks dependencies) but we should mention this somewhere (remark for me).

Reply:
Yes, it's [here](https://github.com/avast/gradle-docker-compose-plugin/pull/351/commits/8c83164c7d3a7db35d6ea268e4e5924cb78e93f2#diff-de1f39ee1c0309d4f56bdaa883c35927bd237df3118f3adab76ce4732fb5f4e6R69)

Reply:
Will this really work? I'm asking because now it looks like it always uses the initial value of `buildBeforePull` (so it doesn't reflect changes by an user later) but I'm in browser only now so maybe the `map` does the trick 🤔 (please pardon possibly stupid question 🙏)

Reply:
Yes, the `map` does the trick. That's lazy property API, and it's should be calculated very late, after user's scripts execution. It's ok question 😉